app/controllers/administrateurs/archives_controller.rb

```diff
@@ -0,0 +1,36 @@
+module Administrateurs
+  class ArchivesController < AdministrateurController
+    before_action :retrieve_procedure, only: [:index, :create]
+    helper_method :create_archive_url
+
+    def index
+      @average_dossier_weight = @procedure.average_dossier_weight
+      @count_dossiers_termines_by_month = Traitement.count_dossiers_termines_by_month(groupe_instructeurs)
+      @archives = Archive.for_groupe_instructeur(groupe_instructeurs).to_a
+    end
+
+    def create
+      type = params[:type]
+      month = Date.strptime(params[:month], '%Y-%m') if params[:month].present?
```

Comment:
```suggestion
      year_month = Date.strptime(params[:year_month], '%Y-%m') if params[:year_month].present?
```

et le mettre en méthode private ?